scm/build/build.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/sh -ex
 export GOOS=linux
-cd $GOPATH/src/github.com/nokia/danm
-glide install --strip-vendor
-go get -d github.com/vishvananda/netlink
-go get github.com/containernetworking/plugins/pkg/ns
-go get github.com/golang/groupcache/lru
+# Force turn on go modules feature becase the project is inside GOPATH
+export GO111MODULE=on
+# Force turn off CGO enables building pure static binaries, otherwise
+# built binary still depends on and dinamically linked against the build
```

Comment:
no? static linker flags are explicitly used for compilation. as far as I can tell it alrady resulted in static libs, without this extra param

Reply:
According to this [article](https://medium.com/@diogok/on-golang-static-binaries-cross-compiling-and-plugins-1aed33499671) and my own experience not enough on centos:

```[user@0200f065c22d danm]$ cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"

[user@0200f065c22d danm]$ CGO_ENABLED=1 go build -o danm ./cmd/danm/; ldd danm
        linux-vdso.so.1 =>  (0x00007fff08fe6000)
        libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f6546478000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f65460aa000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f6546694000)
[user@0200f065c22d danm]$ CGO_ENABLED=0 go build -o danm ./cmd/danm/; ldd danm
        not a dynamic executable```

Reply:
I just realived that I have even forgot to specify the `static` flag in the example and it still resulted in a static binary. My opinion is to explicitly disable CGO because of this 'anomaly'. Would you agree?

Reply:
ok

Reply:
Done